Cargo.toml

```diff
@@ -35,11 +39,14 @@ serde_json = { version = "1.0", features = [
 sha3 = "0.10.1"
 starknet_api = { workspace = true }
 starknet-crypto = "0.5.1"
-thiserror = "1.0.32"
+thiserror = { workspace = true }
 mimalloc = { version = "0.1.29", default-features = false, optional = true }
 hex = "0.4.3"
 anyhow = "1.0.66"
 once_cell = "1.17.1"
+starknet = { workspace = true }
+base64 = { version = "0.21.0", default-features = false, features = ["alloc"] }
+flate2 = "1.0.25"
```

Comment:
Why are all these new dependencies needed for the `starknet_in_rust` crate? Shouldn't those be specific to the RPC client?

Reply:
:f: I totally forgot to remove it. Those aren't needed at all.